internal/pkg/scaffold/constants.go

```diff
@@ -35,5 +35,6 @@ const (
 	DeployDir      = "deploy"
 	OLMCatalogDir  = DeployDir + filePathSep + "olm-catalog"
 	CRDsDir        = DeployDir + filePathSep + "crds"
+	HelmChartsDir  = "helm-charts"
```

Comment:
The reason that I removed it from `internal/pkg/scaffold/helm/chart.go` is that `internal/util/projutil/project_util.go` was importing the entire `internal/pkg/scaffold/helm` package for that one value, and the other additions from this PR in that package were causing a bunch of dep issues.

@estroz's https://github.com/operator-framework/operator-sdk/pull/1297 PR looks like it solves the issue since it declares and uses a separate `helmChartsDir` constant for use in `internal/util/projutil`.

I'll take another look at this to see if I can resolve the dep-related issues without moving `HelmChartsDir` to `constants.go`

Reply:
I don't think there's a way to resolve `projutil` dependency issues aside from copying code to a scaffold package utilities sub-package. `projutil` is meant to be used by the whole project, so it should not import any packages from elsewhere in the SDK.

Reply:
@estroz Agreed. I think we should review and merge #1297, and then I'll merge those changes into this PR and revert fc556e81284d816b24f78767e23321d8d61e15d7